docs/README.md

```diff
@@ -17,3 +17,20 @@ This implementation is compliant with [SSB HTTP Authentication](https://github.c
 A summary can be seen in the following chart:
 
 ![Chart](./login-chart.png)
+
+# HTTP Hosting
+
+We currently assume a standard HTTPS server in front of go-ssb-room to facilitate TLS termination and certificate management. This should be possible with most modern HTTP servers since it's a pretty standard practice, known as [reverse proxying](https://en.wikipedia.org/wiki/Reverse_proxy).
+
+Two bits of rational:
+
+1) People usually want to have more then one site on their server. Put differently we could have LetsEncrypt inside the go-ssb-room server but it would have to listen on port :443, blocking the use of other domains on the same IP.
```

Comment:
```suggestion
1) People usually want to have more than one site on their server. Put differently, we could have [LetsEncrypt](https://letsencrypt.org/) inside the go-ssb-room server but it would have to listen on port :443—blocking the use of other domains on the same IP.
```

fixing typo and then just doing some extras while im at it